### PR TITLE
sumedh/Mar-28

### PR DESCRIPTION
**Pull request by sumedh-mohod**
validation

### CODE REVIEW

File: resources/views/components/employment-form.blade.php [hunk 1]
```diff
@@ -113,3 +113,59 @@ class="form-select form-control @error('employment_status') is-valid @enderror"
...
+                            stringLength: {
+                                min: 4,
+                                message: 'employer name must be more than 4 characters long',
+                            },
+                        },
+                    },
+                    employer_address: {
+                        validators: {
+                            notEmpty: {
+                                message: 'employer address is required',
```
Comment (ithqindia) [inline resources/views/components/employment-form.blade.php:137]: E

File: resources/views/components/employment-form.blade.php [hunk 2]
```diff
@@ -113,3 +113,59 @@ class="form-select form-control @error('employment_status') is-valid @enderror"
...
+                        },
+                    },
+                    job_title: {
+                        validators: {
+                            notEmpty: {
+                                message: 'Aggregates is required',
+                            },
+                            stringLength: {
+                                min: 4,
+                                message: 'employer name must be more than 4 characters long',
```
Comment (ithqindia) [inline resources/views/components/employment-form.blade.php:148]: Wrong message

**Review by ithqindia — CHANGES_REQUESTED** [review]

File: resources/views/components/employment-form.blade.php [hunk 3]
```diff
@@ -125,35 +125,32 @@ class="form-select form-control @error('employment_status') is-valid @enderror"
...
                     },
                     employer_address: {
                         validators: {
                             notEmpty: {
                                 message: 'Employer address is required',
                             },
+                            stringLength: {
+                                min: 10,
+                                max:255,
+                                message: 'Employer address must be more than 5 characters long and less than 255 characters',
```
Comment (manii283) [inline resources/views/components/employment-form.blade.php:138]: change the msessage

File: resources/views/components/employment-form.blade.php [hunk 4]
```diff
@@ -40,9 +40,9 @@
     <div class="col-6">
         <label for="employment_status" class="form-label">Employment status<span style=color:red>*</span></label>
         <select aria-label="Default select example"
-            class="form-select form-control @error('employment_status') is-valid @enderror" name="employment_status"
+            class="form-select form-control @error('employment_status') is-invalid @enderror" name="employment_status"
```
Comment (ithqindia) [inline resources/views/components/employment-form.blade.php:43]: Please explain this change.

**Review by ithqindia — APPROVED** [review]

**Comment by ithqindia** [comment]
Merged locally

**Pull request closed without merging**